test/new-e2e/npm/ec2_1host_test.go

```diff
@@ -0,0 +1,172 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package npm
+
+import (
+	"errors"
+	"math"
+	"os"
+	"testing"
+	"time"
+
+	agentmodel "github.com/DataDog/agent-payload/v5/process"
+	krpretty "github.com/kr/pretty"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/params"
+	"github.com/DataDog/test-infra-definitions/components/datadog/agentparams"
+	"github.com/cenkalti/backoff/v4"
+	"github.com/stretchr/testify/require"
+)
+
+const NPMsystemProbeConfig = `
+network_config:
+  enabled: true
+`
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
This is opinionated, I prefer using `go:embed`, advantage being you can have a yaml file and have it indented properly, avoiding nasty in-code yaml issues. 
Example: https://github.com/DataDog/datadog-agent/blob/main/test/new-e2e/ndm/snmp/snmpTestEnv.go#L34-L35